packages/components/src/combobox-control/styles.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+/**
+ * Internal dependencies
+ */
+import { Flex } from '../flex';
+import { space } from '../ui/utils/space';
+
+const deprecatedDefaultSize = ( { __next36pxDefaultSize } ) =>
+	! __next36pxDefaultSize &&
+	css`
+		height: 30px;
```

Comment:
Just to understand — prior to the changes from this PR, is `ComboBox` height "computed" from the sum of its intrinsic input height + paddings/borders ?